lively.project/project.js

```diff
@@ -218,7 +218,17 @@ export class Project {
     loadedProject.url = url;
     loadedProject.configFile = resource(address.join('package.json').url);
     if (!onlyLoadNotOpen) {
+
       loadedProject.gitResource = await resource('git/' + await defaultDirectory()).join('..').join('local_projects').join(fullName).withRelativePartsResolved().asDirectory();
+      if (await loadedProject.gitResource.hasRemote()) {
+          const remoteURL = await loadedProject.gitResource.getRemote();
+          const userTokenInRemoteURL = remoteURL.match(/(gho_.*)@/)[1];
```

Comment:
This will crash once we introduce a `null` user token due to a non logged in user opening a project.